### PR TITLE
Style for navbar and editor

### PR DESCRIPTION
**Pull request by erizocosmico**
Depends on #5, which also depends on #2.

screenshot_2017-06-26_14 10 44

Right panel is unstyled yet.

Theme for the code editor is solarized light with another clearer bg, which is the one whose color scheme fits more the babelfish color scheme.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -73,6 +87,7 @@ export default class App extends Component {
           actualLanguage={actualLanguage}
           onLanguageChanged={e => this.onLanguageChanged(e)}
           onRunParser={e => this.onRunParser(e)}
+          userHasTyped={userHasTyped}
```
Comment (dpordomingo) [inline src/App.js]: I'd preffer a `dirty` state instead of `userHasTyped`, but not prio at all.

File: src/components/Header.js [hunk 2]
```diff
@@ -70,12 +147,12 @@ export function DriverCode({ languages, selectedLanguage, actualLanguage }) {
...
   return (
     <DriverCodeBox>
-      <img src={githubIcon} alt='GitHub' />
-      <DriverCodeText 
+      <DriverCodeLink
         href={languages[driver].url} 
         target='_blank'>
-        Driver Code
-      </DriverCodeText>
+        <DriverCodeIcon src={githubIcon} alt='GitHub' />
```
Comment (dpordomingo) [inline src/components/Header.js]: alt tag should describe the link target in "linked images"
example: "_Sourcecode of the {languages[driver].name} driver_"

File: src/index.js [hunk 3]
```diff
@@ -11,6 +11,11 @@ injectGlobal`
   * {
     box-sizing: border-box;
   }
+
+  body, html {
+    font-size: 16px;
+    font-family: 'Roboto', sans-serif;
```
Comment (dpordomingo) [inline src/index.js]: I think the main typeface is `Lato` as said at https://github.com/bblfsh/artwork/issues/1#issuecomment-309639525

Comment (erizocosmico) [inline src/index.js]: Ugh, wasn't there an uglier typeface to use? Nvm, I'll change the text to Lato and the code to Inconsolata.

**Comment by erizocosmico** [comment]
Done @dpordomingo

**Comment by erizocosmico** [comment]
ping @Serabe

**Comment by erizocosmico** [comment]
Closing in favor of #8

**Pull request closed without merging**